scripts/vRealize/main.tf

```diff
@@ -0,0 +1,31 @@
+terraform {
+ required_providers {
+    vra = {source = "vmware/vra"}
+ }
+ required_version = ">= 0.13"
+}
+
+provider "vra" {
+  url           = var.vra_url
+  refresh_token = var.vra_refresh_token
+  insecure      = true
+}
+
+data "vra_project" "this" {
+  name = var.vra_project
+}
+
+data "vra_catalog_item" "this" {
+  name = var.vra_catalog_item
+}
+
+resource "vra_deployment" "names" {
+  count           = length(var.vm_names)
+  name            = "CICD-${var.vm_names[count.index]}"
```

Comment:
Lets not make `CICD` hardcoded. So that other teams can also use this framework

Reply:
Done